backend/apps/matching/views/__init__.py

```diff
@@ -4,6 +4,7 @@
 from .participant_profile_edit import ParticipantInfoUpdateView  # noqa
 from .participant_signup import ParticipantSignup  # noqa
 from .profile_dashboard_redirect import ProfileDashboardRedirect  # noqa
+from .staff_approvals import ApproveParticipantsView  # noqa
```

Comment:
OOC: why is # noqa needed here?

Reply:
bc we need to import these into the init if we want to make them available via `views.ApproveParticipants...`
and flake sees them as unused import, hence we need to ignore their warning